ui/components/search-filter-view.jsx

```diff
@@ -1,26 +1,58 @@
 import React from 'react';
+import {Collapse} from 'react-collapse';
 
-export default function SearchFilterView(
-  { filterControls, pageContent, selectedFilters, tabs }) {
-  return (
-    <div className="clearfix">
-      <div className="sidebar col col-2 p2 no-print">
-        Search and filter
-        {filterControls}
-      </div>
-      <div className="main col col-10 pl4 border-left max-width-3">
-        <div className="tab-container no-print">
-          <span className="mr4">View:</span>
-          <ul className="organize-tabs list-reset inline-block">
-            {tabs}
-          </ul>
+export default class SearchFilterView extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {isOpened: false};
+    this.openFilters = this.openFilters.bind(this);
+  }
+
+  arrowClasses() {
+    const defaultClass = "open-close-indicator";
+    const openClass = defaultClass + " open";
+
+    return (this.state.isOpened ? openClass : defaultClass);
+  }
+
+  openFilters() {
+    this.setState(
+      {isOpened: !this.state.isOpened}
+    );
+  }
+
+  render() {
+    return (
+      <div className="clearfix">
+        <a className="mobile-filters" onClick={this.openFilters} role="link">
+          Edit Filters
+          <div className={this.arrowClasses()} />
+        </a>
+        <Collapse isOpened={this.state.isOpened} fixedHeight={180}>
+          <div className="sidebar sm-col sm-col-12 md-hide lg-hide p2 no-print">
+            {this.props.filterControls}
+            <div className="show-results p1 rounded" role="link" onClick={this.openFilters}>Show results</div>
+          </div>
+        </Collapse>
```

Comment:
Does it make sense to move the "Edit Filters" and `Collapse` into a separate container + class, e.g. `MobileFilters`? They're pretty self-contained with regards to state.